apps/veyfi/components/VoteTab.tsx

```diff
@@ -0,0 +1,81 @@
+import {useState} from 'react';
+import Link from 'next/link';
+import {useTransaction} from '@veYFI/hooks/useTransaction';
+import * as VotingEscrowActions from '@veYFI/utils/actions/votingEscrow';
+import {validateAddress, validateNetwork} from '@veYFI/utils/validations';
+import {Button} from '@yearn-finance/web-lib/components/Button';
+import {useWeb3} from '@yearn-finance/web-lib/contexts/useWeb3';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+import {Input} from '@common/components/Input';
+
+import type {ethers} from 'ethers';
+import type {ReactElement} from 'react';
+import type {TAddress} from '@yearn-finance/web-lib/utils/address';
+
+function VoteTab(): ReactElement {
+	const [delegateAddress, set_delegateAddress] = useState('');
+	const {provider, address, isActive, chainID} = useWeb3();
+	const [delegateVote, delegateVoteStatus] = useTransaction(VotingEscrowActions.delegateVote);
+
+	const web3Provider = provider as ethers.providers.Web3Provider;
+	const userAddress = address as TAddress;
+    
+	const {isValid: isValidNetwork} = validateNetwork({supportedNetwork: 1, walletNetwork: chainID});
+	const {isValid: isValidDelegateAddress, error: delegateAddressError} = validateAddress({address: delegateAddress});
+
+	const executeDelegateVote = (): void => {
+		if (!userAddress || !isValidDelegateAddress) {
+			return;
+		}
+		delegateVote(web3Provider, userAddress, toAddress(delegateAddress));
+	};
+
+	return (
+		<div className={'grid grid-cols-1 gap-6 md:grid-cols-2 md:gap-16'}>
+			<div className={'col-span-1 grid w-full gap-8'}>
+				<div className={'md:min-h-[104px]'}>
+					<h2 className={'m-0 text-2xl font-bold'}>
+						{'Vote for Gauge'}
+					</h2>
+					<div className={'mt-6 text-neutral-600'} >
+						<p>{'Vote to direct future YFI rewards to a particular gauge.'}</p>
+						<br />
```

Comment:
😲 we should add gap in the parent `div` :)

Reply:
We shouldn't use that, `flex-col` definitely the way to go here

Reply:
Is the above the right commit? Still see the `<br />` there :)

Reply:
Ahh this github is not letting authors know which line the reviewer clicked on. I definitely clicked on the <br /> line :)

> since its standard line break?

There's no line break there, just 2 paragraphs, if it was `<p>foo<br/>bar</p>` then it would be different